src/implementations/vanilla/components/gallery/_gallery.scss

```diff
@@ -101,15 +108,19 @@ $_description-color: map.get(theme.$color, 'white-100');
 }
 
 .ecl-gallery__info {
-  background-color: map.get(theme.$color, 'grey-10');
+  color: $_gallery-info-color;
   float: right;
-  font: map.get(theme.$font, 'xs');
-  padding: map.get(theme.$spacing, 'xs') map.get(theme.$spacing, 's');
+  font: $_gallery-info-font;
+  padding: map.get(theme.$spacing, 'm') map.get(theme.$spacing, 's');
+}
+
+/* stylelint-disable-next-line plugin/selector-bem-pattern */
+[data-ecl-gallery-all] {
```

Comment:
we should use a proper css class here instead of targeting the data-* attribute.
If there is none on the element, that's the good time to add one